frontend/src/views/Auth/SecurityQuestions/SecurityQuestions.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import "./SecurityQuestions.scss";
+import NavBar from "../../../core/components/NavBar";
+import { Button, Container, Col, Row, Form } from "react-bootstrap";
+import "./SecurityQuestions.scss";
+
+class SecurityQuestions extends Component {
+  // TODO: Integrate with backend
+  state = {
+    securityQuestions: [
+      "What was your childhood nickname?",
+      "In what city did you meet your spouse/significant other?",
+      "What is the name of your favorite childhood friend?",
+      "What street did you live on in third grade?",
+      "What is the middle name of your youngest child?",
+      "What is your oldest sibling's middle name?",
+      "What school did you attend for sixth grade?",
+      "What is your oldest cousin's first and last name?",
+      "What was the name of your first stuffed animal?",
+      "In what city or town did your mother and father meet?",
+      "Where were you when you had your first kiss?",
+      "What is the first name of the boy or girl that you first kissed?",
+      "What was the last name of your third grade teacher?",
+      "In what city does your nearest sibling live?",
+      "What is your maternal grandmother's maiden name?",
+      "In what city or town was your first job?",
+      "What is the name of the place your wedding reception was held?"
+    ]
```

Comment:
We need create a process to insert this data on db.
And then that backend send back these questions.

Reply:
Yes, this was made only as a placholder until the backend was ready

Reply:
Ok